src/poliastro/coordinates.py

```diff
@@ -130,3 +131,36 @@ def inertial_body_centered_to_pqw(r, v, source_body):
     v_pqw = (np.array([-sin(nu), (ecc + cos(nu)), 0]) * sqrt(k / p)).T * u.km / u.s
 
     return r_pqw, v_pqw
+
+
+def change_frame(orbit, frame_orig, frame_dest):
```

Comment:
I'd rather call this `transform`, to be consistent with Astropy method `transform_to` :)